src/components/Avatar/__stories__/avatar.stories.mdx

```diff
@@ -0,0 +1,205 @@
+import { useState, useCallback, useMemo } from "react";
+import Avatar from "../Avatar";
+import { createComponentTemplate } from "../../../storybook/functions/create-component-story";
+import { ArgsTable, Story, Canvas, Meta } from "@storybook/addon-docs";
+import { Calendar, Remove, Add, Check } from "../../Icon/Icons";
+import { TOOLTIP, LABEL, CHIPS } from "../../../storybook/components/related-components/component-description-map";
+import "./avatar.stories.scss";
```

Comment:
please move the scss import to be the last import in the file (that's our convention)